### PR TITLE
gitdb: handle error better

### PR DESCRIPTION
**Pull request by dbentley**
scootapi: fix a watch_job bug

### CODE REVIEW

File: scootapi/client/watch_job_cmd.go [hunk 1]
```diff
@@ -75,12 +75,17 @@ func PrintJobStatus(jobStatus *scoot.JobStatus) {
 	for taskId, taskStatus := range jobStatus.TaskStatus {
 		fmt.Printf("\tTask %s {\n", taskId)
 		fmt.Printf("\t\tStatus: %s\n", taskStatus.String())
-		runStatus := jobStatus.TaskData[taskId]
-		if runStatus.OutUri != nil {
-			fmt.Printf("\t\tStdout: %v\n", *runStatus.OutUri)
-		}
-		if runStatus.ErrUri != nil {
-			fmt.Printf("\t\tStderr: %v\n", *runStatus.ErrUri)
+		runStatus, ok := jobStatus.TaskData[taskId]
```
Comment (jasonbs10) [inline scootapi/client/watch_job_cmd.go]: Are we fine with task data being null instead of PENDING?

**Comment by dbentley** [comment]
Yes, we should fix the server, *and* we should fix the client to not error, because who knows what happens on the server?

File: scootapi/client/watch_job_cmd.go (hunk 1 above)
Comment (dbentley) [inline scootapi/client/watch_job_cmd.go]: Fixed that up.

**Comment by jasonbs10** [comment]
Hmm, trick question? *We* know what's supposed to happen on the server. I'd say this is panic()-worthy but we probably do want to err towards best effort.

**Pull request merged**